### ISSUE TITLE
Save tags automatically

### ISSUE THREAD

**Issue by invenio-developers** [inveniosoftware/invenio#213]
Originally by _[vengmark (@l0b0)](https://github.com/l0b0)_ on 2010-07-13

Pros:
- Virtually no interaction necessary to save tags
  - "Virtually" since it would be very slow if we save after each keypress, so when do we save? We'll need to take into account normal typing, middle-click paste, Ctrl-V paste, right-click paste, and drag-n-drop across browsers and platforms. Does anyone know of a simple way to do this?

Cons:
- Users are not used to this type of interaction
  - Unless it is completely clear when tags are saved, they may navigate away and wonder why their last modification was not saved.
  - Users expect a Save / Submit button at the end of each form, and may assume that the form is defect without trying it.
- Undo would have to be implemented separately (ouch!). On the web, people "know" that navigating away from the page leaves data unsaved.
- In summary, much more complex code and beginner user interaction.

Non-issues:
- Clicking a link navigates away from the form, but since we're not modifying the form on load it will still contain the same data if the user uses the Back button.
